docs/doc/11-integrations/30-file-format-options.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Input File Formats
+---
+
+Databend accepts a variety of file formats as a source where you can load data from with the [COPY INTO command](../14-sql-commands/10-dml/dml-copy-into-table.md) or [Streaming Load API](../11-integrations/00-api/03-streaming-load.md). When you select a file to do that, you need to tell Databend what the file looks like using the following format:
+
+```sql
+FILE_FORMAT = ( TYPE = { CSV | TSV | NDJSON | PARQUET | XML } [ formatTypeOptions ] )
+```
+
+`Type`: Specifies the file format. Must be one of the ones listed above that Databend supports.
+
+`formatTypeOptions`: Includes one or more options to describe other format details about the file. The options vary depending on the file format. See the sections below to find out the available options for each supported file format.
+
+```sql
+formatTypeOptions ::=
+  RECORD_DELIMITER = '<character>'
+  FIELD_DELIMITER = '<character>'
+  SKIP_HEADER = <integer>
+  QUOTE = '<character>'
+  ESCAPE = '<character>'
+  NAN_DISPLAY = '<string>'
+  ROW_TAG = '<string>'
+  COMPRESSION = AUTO | GZIP | BZ2 | BROTLI | ZSTD | DEFLATE | RAW_DEFLATE | XZ | NONE
+```
+
+## CSV Options
+
+Databend accepts CVS files that are compliant with [RFC 4180](https://www.rfc-editor.org/rfc/rfc4180) and is subject to the following conditions:
+
+- A string must be quoted if it contains the character of a [QUOTE](#quote), [ESCAPE](#escape), [RECORD_DELIMITER](#record_delimiter), or [FIELD_DELIMITER](#field_delimiter).
+- No character will be escaped in a quoted string except `Quote`.
+- No space should be left between a `FIELD_DELIMITER` and a `Quote`.
+- A record should not contain a trailing `FIELD_DELIMITER`.
+- A string will be quoted in CSV if it comes from a serialized Array or Struct field.
+- If you develop a program and generate the CSV files from it, Databend recommends using the CSV library from the programing language.
+- Databend does not recognize the files unloaded from MySQL as the CSV format unless the following conditions are satisfied:
+  - `ESCAPED BY` is empty.
+  - `ENCLOSED BY` is not empty.
+:::note
+Files will be recognized as the TSV format if the conditions above are not satisfied. For more information about the clauses `ESCAPED BY` and `ENCLOSED BY`, refer to https://dev.mysql.com/doc/refman/8.0/en/load-data.html.
+:::
+
+### RECORD_DELIMITER
+
+Separates records in an input file.
+
+**Available Values**: `\r`,`\n`, or use a character with the escape char: `\b`, `\f`, `\r`, `\n`, `\t`, `\0`, `\xHH`
```

Comment:
any visible character like `a` or `|` is also ok.

Reply:
the limitation is  "one character" except `\r\n` 
`\b`, `\f`, `\r`, `\n`, `\t`, `\0`, `\xHH` are acceptable escaped form of  "one character"

Reply:
**Available Values**: 
- `\r\n`
- An arbitrary character, such as `a` and `|`.
- A character with the escape char: `\b`, `\f`, `\r`, `\n`, `\t`, `\0`, `\xHH`

Reply:
ok to me.